configs/model/reverse_distillation.yaml

```diff
@@ -6,8 +6,6 @@ model:
       - layer1
       - layer2
       - layer3
-    beta1: 0.5
-    beta2: 0.999
```

Comment:
I think this is the wrong config so I deleted it. Can you please check it?

Reply:
Yeah you are right